### PR TITLE
Added section on GitHub Projects

### PR DESCRIPTION
**Pull request by ghost**
As per Issue #56.

### CODE REVIEW

**Review by abbycabs — APPROVED** [review]
I actually hadn't thought of using the 'Projects' feature as a Roadmap. This is pretty great!

Like @betatim suggested in [the issue](https://github.com/mozilla/open-leadership-training-series/issues/56#issuecomment-274439347), it would be nice to link to a project that is managed this way.

Also, since this is a fairly new feature, I'm not sure if there are any established best practices around Projects. Should Projects be linked from the README or CONTRIBUTING.md? Or will people know to look there (like issues)?

👍 I approve this! With some questions for the future :)

**Comment by ghost** [comment]
I'm working on looking for some examples and I think, it terms of best practices, I think a reference to the repo's Projects should be in both of the markdown files that you mentioned as it relates to both basic information and how to contribute.  It might be something that could be tested out in (maybe) Round 3 of the Leadership Mentoring Program.

**Comment by abbycabs** [comment]
Sounds good! I'll wait for some examples before merging

File: _articles/opening-your-project/start-your-project-roadmap.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ The timeline of tasks is the probably most important part of the Roadmap so as a
     *   Short term - tasks you are working on now
     *   Medium term - tasks that are coming up next
     *   Long term - where your project is going, beyond the next few weeks or months
+    
+### GitHub Projects
+
+Another way to track your project's progress is use to use [GitHub's Projects](https://help.github.com/articles/tracking-the-progress-of-your-work-with-projects/).  This feature allows you create [kanban-style boards](https://en.wikipedia.org/wiki/Kanban_(development)) to manage tasks, take Notes to quickly add tasks or reminders to your project which can be converted to Issues when you're ready.
```
Comment (betatim) [inline _articles/opening-your-project/start-your-project-roadmap.md:40]: I like that you linked to a description of kanban!

**Comment by ghost** [comment]
I found one example and added it.

**Comment by betatim** [comment]
Did you mean to add the text about burn out to this branch? I would prefer to keep them separate. If you would like help with `git` commands to undo this let me know.

**Comment by ghost** [comment]
Yeah, I meant to keep it separate.  Let me fix it.

**Comment by ghost** [comment]
And done.

**Comment by betatim** [comment]
:rocket: thanks!

**Pull request merged**